scripts/redis_crash_handler.py

```
@@ -0,0 +1,1134 @@
+        )
+        response.raise_for_status()
+        self.token = response.json()["jwtToken"]
+    
+    def _get_headers(self):
+        """Get authentication headers"""
+        return {
+            "Content-Type": "application/json",
+            "Authorization": f"Bearer {self.token}"
+        }
+    
+    def get_customer_info(self, service_id: str, environment_id: str, namespace: str) -> CustomerInfo:
```

Ruff (0.14.14)

[warning] Unused method argument: `service_id`

(ARG002)